### DISCUSSION TITLE
Widgets

### DISCUSSION THREAD

**Discussion by Roxo7777** [z-huang/music#1363] [Ideas]
Just bring widgets to put on the home screen, preferably that follow the color of the cover photo of the song, and not the material you.

**Comment by pdimu** [comment]
An option to switch between Monet of cover and Material You would be better in this case

**Reply by Fabri66rc** [reply]
Hello! Can I ask you a question? I am fine with innertune and I was wondering if by chance there is a dedicated widget? Thanks

**Reply by pdimu** [reply]
> Hello! Can I ask you a question? I am fine with innertune and I was wondering if by chance there is a dedicated widget? Thanks
@Fabri66rc 

Unfortunately, there is no widget implemented already

**Comment by MihaiCristianCondrea** [comment]
Waiting for widgets, the best idea :D It will bring my Pixel better aesthetics + easy app access <3

**Comment by pdimu** [comment]
@z-huang I got this screenshot right here from the owner of Oto Music (they showcased how the md3 widgets will be when they roll out in v4.0)
![Screenshot_20240415-114020.png](https://github.com/user-attachments/assets/15147927-beb8-48d0-b57b-fae0f7caba10)
![Screenshot_20240415-113942.png](https://github.com/user-attachments/assets/61e1d20f-4d4d-43b8-9765-a7bd6570a505)

I think my favourite is this miniplayer one
![Screenshot_2024-09-06-11-20-36-688-edit_io.github.pyoncord.app.jpg](https://github.com/user-attachments/assets/56af1ee9-1893-47cc-9b36-63e311cbac71)

**Reply by Roxo7777** [reply]
I particularly prefer it without any evidence of Material You in the widgets, Spotify, Deezer and YouTube Music did it and I didn't like it, I prefer based on the album colors, totally. I know, the app's proposal is from Material You, but honestly, it's ugly.

**Reply by pdimu** [reply]
> I particularly prefer it without any evidence of Material You in the widgets, Spotify, Deezer and YouTube Music did it and I didn't like it, I prefer based on the album colors, totally. I know, the app's proposal is from Material You, but honestly, it's ugly. 
> @Roxo7777 

There is an option to switch between Material You and Monet generation from song art. 
I think another switch (but for the widget, not the entire UI) would work for you, without dragging the other users' experience down

**Comment by Fabri66rc** [comment]
Hi, since yesterday evening Innertune has stopped working. It reads all the songs in my playlists as "error" and goes forward very fast, without ever stopping! There are two things, either it is possessed by an evil soul, or there is something that is not working
________________________________
Da: Dimu ***@***.***>
Inviato: martedì 17 dicembre 2024 12:22
A: z-huang/InnerTune ***@***.***>
Cc: Fabri66rc ***@***.***>; Mention ***@***.***>
Oggetto: Re: [z-huang/InnerTune] Widgets (Discussion #1363)


Hello! Can I ask you a question? I am fine with innertune and I was wondering if by chance there is a dedicated widget? Thanks
@Fabri66rc<https://github.com/Fabri66rc>

Unfortunately, there is no widget implemented already

**Reply by pdimu** [reply]
Could you create a new issue regarding your problem, please?
